### DISCUSSION TITLE
X-Spam to X-Spam-Flag

### DISCUSSION THREAD

**Discussion by chandruv21** [rspamd/rspamd#3802] [Show and tell]
Hi guyzz,

Rspamd is my first experience with it. I've successfully installed rspamd on Debian 10 and integrated it with postfix; now I need to modify the X-Spam header to X-Spam-Flag for spam emails. What is the procedure for altering the header?

Thank you in Advance.

Regards,
Chandrashekar

**Comment by chandruv21** [comment]
Edit /etc/rspamd/local.d/worker-proxy.inc configuration file and change
 X-Spam to X-Spam-Flag

Regards,
Chandrashekar

**Comment by kasak87** [comment]
alternatively you can have both x-spam and x-spam-flags. Just create local.d/milter_headers.conf with:
```
use = ["spam-header"]

routines {
	spam-header {
		header = "X-Spam-Flag";
		value = "YES";
		remove = 0;
	}
}
```

**Reply by ThiefMaster** [reply]
This results in slightly weird and likely unwanted behavior: Mails that get a subject rewrite will ONLY have X-Spam-Flag:YES, while mails that "just" get the spam header will have both.
